### PR TITLE
MVP GUI for Media Grapher

### PR DESCRIPTION
**Pull request by dwang312**
# Pull Request

## Describe your changes
1. Edited README.md to include:
- How to utilize GUI

2. Updated requirements.txt
- Added dependencies to run GUI (PYQT)

3. Introduced GUI for Media Grapher
- GUI is introduced to project to run the Media Grapher Script (``gui.py``)
```bash
python mediagrapher.py [-o OUTPUT] [-a {Canny,Sobel}] [-t LOW HIGH] url
```

## Topic

- [X] Enhancement / New Feature
- [ ] Bug Fix
- [X] Documentation
- [ ] Test
- [ ] Breaking Change

## Supporting Docs
MediaGrapherGUI

[Video URL used](https://www.youtube.com/watch?v=dyVaHHn7j5Y&ab_channel=ItzSlashy)

Result:
image
image

### CODE REVIEW

**Comment by dwang312** [comment]
``mediagrapher.py`` conflict involves the addition of ``thread`` parameter.

**Comment by hMRZQ21** [comment]
Hey David, 
I noticed you're having the same issue I did with the number of `Files Changed`. Git thinks you changed everything. 

I suggest you take out your changes on the side, discard your commits, and then paste them back in.

**Comment by dwang312** [comment]
After discussing with the repo author, the following files should be closely during the PR review.
- ``.pylintrc``
- ``gui.py``
- ``README.md``

**Review by jedrm — APPROVED** [review]
LGTM

**Comment by jedrm** [comment]
@dwang312 There are still some Pylint errors in the code

**Comment by dwang312** [comment]
> @dwang312 There are still some Pylint errors in the code

Current Pylint errors, will look into how to resolve or how to approach this.

```bash
Run pylint $(git ls-files '*.py')
************* Module gui
gui.py:12:0: E0401: Unable to import 'PyQt6.QtGui' (import-error)
gui.py:13:0: E0401: Unable to import 'PyQt6.QtWidgets' (import-error)

```

The test run does list PyQt6 as having been installed.
```bash
Collecting PyQt6==6.6.1 (from -r requirements.txt (line 58))
  Downloading PyQt6-6.6.1-cp38-abi3-manylinux_2_28_x86_64.whl.metadata (2.1 kB)
Collecting PyQt6-Qt6==6.6.1 (from -r requirements.txt (line 59))
  Downloading PyQt6_Qt6-6.6.1-py3-none-manylinux_2_28_x86_64.whl.metadata (534 bytes)
Collecting PyQt6-sip==13.6.0 (from -r requirements.txt (line 60))
  Downloading PyQt6_sip-13.6.0-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.whl.metadata (503 bytes)
  ```

File: .pylintrc [hunk 1]
```diff
@@ -1,641 +1,643 @@
...
+clear-cache-post-run=no
+
+# Load and enable all available extensions. Use --list-extensions to see a list
+# all available extensions.
+#enable-all-extensions=
+
+# In error mode, messages with a category besides ERROR or FATAL are
+# suppressed, and no reports are done by default. Error mode is compatible with
+# disabling specific errors.
+#errors-only=
```
Comment (jedrm) [inline .pylintrc:19]:
```suggestion
errors-only=yes
```

See if enabling this Pylint configuration fixes the issue

**Review by jedrm — CHANGES_REQUESTED** [review]

File: .pylintrc (hunk 1 above)
Comment (dwang312) [reply .pylintrc]: Commit made to reflect this suggestion.

**Review by jedrm — APPROVED** [review]

**Comment by jedrm** [comment]
New PR has been created and can be found here: #37

**Pull request closed without merging**